### PR TITLE
Upgrade asciidoctor and dependencies to latest versions

### PR DESCRIPTION
**Pull request by sapientpants**

### CODE REVIEW

**Comment by dduportal** [comment]
Thanks for your contribution and your patience! I took the freedom to fix conflicts on your PR so it's up to date with latest changes.

Closes #167

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
-FROM alpine:3.12
+FROM alpine:3.12.1
```
Comment (dduportal) [inline Dockerfile:1]: Is there a particular reason for this change?
For information, the Docker Image is rebuilt automatically when the upstream `alpine:3.12` image is updated,
so this change is not needed:

Screenshot 2020-12-10 at 18 31 14

Comment (dduportal) [reply Dockerfile:1]:
```suggestion
FROM alpine:3.12
```

**Review by dduportal — CHANGES_REQUESTED** [review]
Hello @sapientpants , thanks a lot for your contribution and patience!

This PR is ready to be merged except for the `FROM` diretctive change.

Would you mind fixing it please?

File: README.adoc [hunk 2]
```diff
@@ -10,18 +10,18 @@ GitHub renders asciidoctor natively, but DockerHub does not (it needs markdown).
...
 * https://asciidoctor.org/docs/asciidoctor-pdf/[Asciidoctor PDF] 1.5.3
 * https://asciidoctor.org/docs/asciidoctor-epub3/[Asciidoctor EPUB3] 1.5.0.alpha.19
-* https://github.com/asciidoctor/asciidoctor-mathematical[Asciidoctor Mathematical] 0.3.1
+* https://github.com/asciidoctor/asciidoctor-mathematical[Asciidoctor Mathematical] 0.3.4
 * https://asciidoctor.org/docs/asciidoctor-revealjs/[Asciidoctor reveal.js] 4.0.1
 * https://rubygems.org/gems/asciimath[AsciiMath]
 * Source highlighting using http://rouge.jneen.net[Rouge] or https://rubygems.org/gems/coderay[CodeRay] (Pygments not supported in the default Docker image as only Python 3 is available)
 * https://github.com/asciidoctor/asciidoctor-confluence[Asciidoctor Confluence] 0.0.2
-* https://github.com/asciidoctor/asciidoctor-bibtex[Asciidoctor Bibtex] 0.7.1
+* https://github.com/asciidoctor/asciidoctor-bibtex[Asciidoctor Bibtex] 0.80.0
```
Comment (slonopotamus) [inline README.adoc:22]: This should be `0.8.0`, not `0.80.0`

File: README.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ This Docker image provides:
 
 -   [Asciidoctor Confluence](https://github.com/asciidoctor/asciidoctor-confluence) 0.0.2
 
--   [Asciidoctor Bibtex](https://github.com/asciidoctor/asciidoctor-bibtex) 0.7.1
+-   [Asciidoctor Bibtex](https://github.com/asciidoctor/asciidoctor-bibtex) 0.80.0
```
Comment (slonopotamus) [inline README.md:25]: Same problem as in README.adoc.

File: Dockerfile [hunk 4]
```diff
@@ -62,8 +62,9 @@ RUN apk add --no-cache --virtual .rubymakedepends \
     asciimath \
     "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}" \
     "asciidoctor-revealjs:${ASCIIDOCTOR_REVEALJS_VERSION}" \
+    bigdecimal \
```
Comment (jirutka) [inline Dockerfile:65]: If bigdecimal is needed, install the version shipped with ruby, it's provided by Alpine package ruby-bigdecimal.

**Comment by sapientpants** [comment]
The outstanding change requests should all be taken care of with the latest commit.

**Review by dduportal — APPROVED** [review]
LGTM, thanks for this contribution ! 👍

**Pull request merged**